tests/seed_isort_config_test.py

```diff
@@ -95,12 +99,72 @@ def test_integration_extra_file(tmpdir):
         assert tmpdir.join('.isort.cfg').read() == expected
 
 
-def test_integration_no_config(tmpdir, capsys):
+@pytest.mark.parametrize(
+    'initial_filesystem,expected_filesystem',
+    [
+        ({}, {'.isort.cfg': '[settings]\nknown_third_party=cfgv\n'}),
```

Comment:
the parametrize is cute, but I'd rather see these as separate named tests so it's obvious what's going on.  I'd name this one `test_creates_isort_cfg_when_none_exist`

This produces an additional issue however, this change won't properly fail the commit (it does not modify any of the tracked files).  I think this should be an error case